diff --git a/node_modules/next/image-types/global.d.ts b/node_modules/next/image-types/global.d.ts
index a41a6c6..8923d18 100644
--- a/node_modules/next/image-types/global.d.ts
+++ b/node_modules/next/image-types/global.d.ts
@@ -15,12 +15,7 @@ declare module '*.png' {
 }
 
 declare module '*.svg' {
-  /**
-   * Use `any` to avoid conflicts with
-   * `@svgr/webpack` plugin or
-   * `babel-plugin-inline-react-svg` plugin.
-   */
-  const content: any
+  const content: React.FC<React.SVGProps<SVGElement>>
 
   export default content
 }
