diff --git a/node_modules/next/image-types/global.d.ts b/node_modules/next/image-types/global.d.ts
index 1a1c964..6b8d02f 100644
--- a/node_modules/next/image-types/global.d.ts
+++ b/node_modules/next/image-types/global.d.ts
@@ -15,12 +15,9 @@ declare module '*.png' {
 }
 
 declare module '*.svg' {
-  /**
-   * Use `any` to avoid conflicts with
-   * `@svgr/webpack` plugin or
-   * `babel-plugin-inline-react-svg` plugin.
-   */
-  const content: any
+  export type SVG = React.FC<React.SVGProps<SVGElement>>
+
+  const content: SVG
 
   export default content
 }
