diff --git a/node_modules/undici/lib/web/fetch/webidl.js b/node_modules/undici/lib/web/fetch/webidl.js
index 46850e1..f8da761 100644
--- a/node_modules/undici/lib/web/fetch/webidl.js
+++ b/node_modules/undici/lib/web/fetch/webidl.js
@@ -1,7 +1,7 @@
 'use strict'
 
 const { types, inspect } = require('node:util')
-const { markAsUncloneable } = require('node:worker_threads')
+const { markAsUncloneable, MessagePort } = require('node:worker_threads')
 const { toUSVString } = require('../../core/util')
 
 const UNDEFINED = 1
@@ -71,7 +71,7 @@ webidl.argumentLengthCheck = function ({ length }, min, ctx) {
   if (length < min) {
     throw webidl.errors.exception({
       message: `${min} argument${min !== 1 ? 's' : ''} required, ` +
-               `but${length ? ' only' : ''} ${length} found.`,
+        `but${length ? ' only' : ''} ${length} found.`,
       header: ctx
     })
   }
@@ -132,7 +132,7 @@ webidl.util.TypeValueToString = function (o) {
   }
 }
 
-webidl.util.markAsUncloneable = markAsUncloneable || (() => {})
+webidl.util.markAsUncloneable = markAsUncloneable || (() => { })
 
 // https://webidl.spec.whatwg.org/#abstract-opdef-converttoint
 webidl.util.ConvertToInt = function (V, bitLength, signedness, opts) {
